docs/neural_modeling/plot_06_calcium_imaging.py

```diff
@@ -103,26 +126,26 @@
 coupling_basis = nmo.basis.RaisedCosineBasisLog(3, mode="conv", window_size=10)
 
 # %%
-# We need to make sure the design matrix will be full-rank by applying identifiability constraints
-# to the Cyclic Bspline
-heading_basis.identifiability_constraints = True
+# We need to make sure the design matrix will be full-rank by applying identifiability constraints to the Cyclic Bspline, and then combine the bases (the resturned object will be an `AdditiveBasis` object).
 
-# Here we combine the basis. The returned object is an `AdditiveBasis` object.
+heading_basis.identifiability_constraints = True
 basis = heading_basis + coupling_basis
 
+
 # %%
 # ## Gamma GLM
 #
-# Since the transients are non-negative, we will use a Gamma distribution from `nemos` with a soft-plus non linearity.
-#
+# Until now, we have been modeling spike trains, and have used a Poisson distribution for the observability model. With calcium traces, things are quite different: we no longer have counts but continuous signals, so the Poisson is no longer appropriate. We cannot use a Gaussian because the calcium traces are non-negative. To satisfy these constraints, we will use a Gamma distribution from `nemos` with a soft-plus non linearity.
+
```

Comment:
Following up on your comment, we can add a note about the non-linearity. something of this kind
```suggestion
!!! note "Non-linearity"
    Different option are possible. With a soft-plus we are assuming an "additive" effect the predictors, while an exponential non-linearity assumes multiplicative effects. Deciding which firing rate model works best is an empirical question. You can fit different configurations to see which one capture best the neural activity.
```

Reply:
These are all helpful feedback, I will incorporate asap!